### DISCUSSION TITLE
Confluence Cloud?

### DISCUSSION THREAD

**Discussion by towi** [bsorrentino/maven-confluence-plugin#259] [Q&A] [answered]
Is there any chance to upload my maven projects documentation not to a Confluence Server but to the new [Confluence Cloud](https://developer.atlassian.com/cloud/confluence/rest/intro/)? Or maybe it is alreadly possible, but I don't see it?

**Comment by qwazer** [comment, marked as answer]
Hi @towi.
 Just setup connection to your confluence instance. It is not matter is it on-premise or cloud.
